pkg/daemon/ceph/client/mirror_health.go

```diff
@@ -134,7 +134,7 @@ func updatePoolStatusMirroring(c *mirrorChecker, mirrorStatus *cephv1.MirroringS
 	blockPool := &cephv1.CephBlockPool{}
 	if err := c.client.Get(c.clusterInfo.Context, c.namespacedName, blockPool); err != nil {
 		if kerrors.IsNotFound(err) {
-			logger.Debug("CephBlockPool %q resource not found for updating the mirroring status, ignoring.", blockPool)
+			logger.Debug("CephBlockPool %v resource not found for updating the mirroring status, ignoring.", blockPool)
```

Comment:
We generally just need to log the name of the resource, rather than `%v`
```suggestion
			logger.Debug("CephBlockPool %q resource not found for updating the mirroring status, ignoring.", c.namespacedName)
```